### ISSUE TITLE
Allow to specify options for :source-highlighter: rouge

### ISSUE THREAD

**Issue by aka-demik** [asciidoctor/asciidoctor#1785]
[Options list](https://github.com/jneen/rouge#full-options)
For example:

``` asciidoc
:rouge-formatter-options: option1: value; option2: value
:rouge-lexer-options: ...
```

**Comment by mojavelinux** [comment]
First, we need to actually support Rouge :) See #1040.

**Comment by aka-demik** [comment]
I have seen #1040 :innocent:. And I'm looking forward to it.

**Comment by mojavelinux** [comment]
#1040 is now resolved (and will be available in Asciidoctor 2.0).

The syntax highlighter is now an extension point. Therefore, if you want to add additional features such as what you're proposing, you are free to supply your own implementation. Putting these directly in core adds too much complexity to maintain and is thus out of scope.
